packages/forms/src/elements/Input.tsx

```diff
@@ -42,7 +42,7 @@ export const Input = React.forwardRef<HTMLInputElement, IInputProps>(
     }
 
     if (fieldContext) {
-      combinedProps = fieldContext.getInputProps(combinedProps, { isDescribed: true });
```

Comment:
I found that the `isDescribed` option is `true` on multiple input components including `Input`. I found it was safe to remove after `getInputProps` was given a default value for the second function parameter. Are there any concerns with removing?